### ISSUE TITLE
In version 1.1.0 all types have become private

### ISSUE THREAD

**Issue by venil7** [xemwebe/yahoo_finance_api#9]
Quote, YahooError, etc and others have become private, making impossible to use them in fn dеfinitions

```Rust
pub fn test(a: &yahoo_finance_api::quotes::YResponse) {}
    |                                      ^^^^^^ private module

```

**Comment by xemwebe** [comment]
Oh yes, thanks for the hint, fill fix that soon.

**Comment by xemwebe** [comment]
Just pushed/published and update

**Comment by venil7** [comment]
confirming, this is fixed in 1.1.3 
thank you!
